charts/spire/charts/spire-server/templates/roles.yaml

```diff
@@ -18,6 +18,19 @@ rules:
     verbs:
       - get
       - patch
+  {{- if and .Values.upstreamAuthority.certManager.enabled .Values.upstreamAuthority.certManager.rbac.create }}
+  - apiGroups: ["cert-manager.io"]
+    resources:
+      - certificates
+      - certificaterequests
+    verbs:
+      - list
+      - get
+      - create
+      - delete
```

Comment:
Should it be possible to delete certificates by spire-server?

Reply:
not sure... Now that we have a test suite, I could try removing it and see what happens.

Reply:
Probably we should verify which exact permissions the plugin requires, so we can narrow it down as far as possible, without breaking functionality.

Reply:
It has to get an intermediate ca cert from cert-manager in order to give the oidc pod a cert. So if thats all working in tests, I think we're ok.

It does seem to work without certificate permissions, so I think this has been fixed?